cocos/ui/UIPageView.h

```diff
@@ -76,7 +76,15 @@ class CC_GUI_DLL PageView : public Layout
     enum class TouchDirection
     {
         LEFT,
-        RIGHT
+        RIGHT,
+        UP,
```

Comment:
my mistake, I will add them. But actually developer don't need to care about it.